py_gtfs_rt_ingestion/lib/s3_utils.py

```diff
@@ -171,14 +171,15 @@ def move_s3_objects(files: List[str], destination: str) -> None:
     process_logger = ProcessLogger(
         "move_s3_objects",
         pool_size=pool_size,
-        destination=destination,
+        to_bucket=to_bucket,
         file_count=len(files),
     )
     process_logger.log_start()
 
-    with ThreadPoolExecutor(max_workers=pool_size) as executor:
-        for filename in files:
-            executor.submit(_move_s3_object, filename, destination)
+    with Pool(pool_size, initializer=_init_process_session) as pool:
+        pool.starmap(
+            _move_s3_object, [(filename, to_bucket) for filename in files]
+        )
```

Comment:
initialize each process with `_init_process_session` which saves a boto Session in the process dictionary object. this might not be the best way to save the Session, but it seems to work.